esp-hal/src/soc/esp32/psram.rs

```diff
@@ -12,50 +12,75 @@
 //!
 //! The `PSRAM` module is accessed through a virtual address, defined as
 //! `PSRAM_VADDR`. The starting virtual address for the PSRAM module is
-//! 0x3F800000. The `PSRAM` module size depends on the configuration specified
-//! during the compilation process. The available `PSRAM` sizes are `2MB`,
-//! `4MB`, and `8MB`.
-//!
-//! NOTE: If you want to use `PSRAM` on `ESP32` or `ESP32-S3`, it'll work only
-//! in `release` mode.
+//! 0x3F800000.
 
 /// The starting virtual address of the PSRAM (Pseudo-SRAM) region in memory.
 pub const PSRAM_VADDR_START: usize = 0x3F800000;
 
+/// The total size of the PSRAM (Pseudo-SRAM) in bytes.
+static mut PSRAM_BYTES: u32 = 0;
```

Comment:
We should probably unify some of this somewhere so we don't duplicate a bunch of things for each soc